2024/survey.yaml

```
@@ -356,6 +356,19 @@ questions:
       - Wiki (unofficial)
       - Blog posts
       - Others
+  - prompt: If you want to contribute or have contributed in the past to Nixpkgs, please select all that apply:
+    type: multiple
+    choices:
+      - I don't plan to contribute.
+      - Everything I want to contribute to is already being worked on and I don't feel the need to get involved.
+      - I would like to contribute but I don't know how to get started.
+      - I tried to contribute something but found it hard to get started.
```

This indeed feels redundant. How about this?
```suggestion
      - I would like to contribute but I don't know how to get started.
      - I tried to contribute something but I got stuck in the process.
```

---

Perhaps reuse "started" to reinforce the difference between the two options?

```suggestion
      - I would like to contribute but I don't know how to get started.
      - I tried to contribute something but got stuck after I started.
```